cmd/meroxa/turbine/utils.go

```diff
@@ -55,6 +55,9 @@ type ApplicationResource struct {
 	Destination bool   `json:"destination"`
 	Collection  string `json:"collection"`
 }
+type HTTPClient interface {
+	Do(req *http.Request) (*http.Response, error)
```

Comment:
💯❗

Reply:
oh i didn't end up using this... i rejected this implementation of the unit test